runHarness/ComputeResources/KubernetesCluster.pm

```diff
@@ -852,7 +852,7 @@ sub kubernetesTopPodAllNamespaces {
 				$logger->info("kubernetesTopPodAllNamespaces top pod failed: $cmdFailed");
 			}
 			my $time;
-			$cmd = "date +%H:%M";
+			$cmd = "date -u";
```

Comment:
Should the date command in kubernetesTopNode be similarly changed?

Reply:
Yes.  Good point.  I needed this change to simplify matching up timestamps in the logs with these times